src/main/groovy/com/morpheusdata/upcloud/UpcloudProvisionProvider.groovy

```diff
@@ -99,7 +103,11 @@ class UpcloudProvisionProvider extends AbstractProvisionProvider implements Work
 	 */
 	@Override
 	Collection<StorageVolumeType> getRootVolumeStorageTypes() {
-		Collection<StorageVolumeType> volumeTypes = []
+		Collection<StorageVolumeType> volumeTypes = [
+			new StorageVolumeType(
+				code:'upcloudVolume'
```

Comment:
for upcloud specific data like `upcloudVolume` the whole object will need populated.  For items like line 82:
```
new OptionType(
    code:'provisionType.general.noAgent'
)
```
Just the code is sufficient since the data is created by core seeds and shared across multiple integrations.